devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -569,7 +529,7 @@ private Map getXyzResult(results, description) {
 private getManufacturerCode() {
 	if (device.getDataValue("manufacturer") == "SmartThings") {
 		return "0x110A"
-	} else {
+	} else if (manufacturer == "CentraLite") {
```

Comment:
I'm not sure this is a good idea. If a device was joined a long time ago it may not have manufacturer set in device data. I think it's safer to leave it as it was.